apollo-adminservice/src/main/config/application-github.properties

```diff
@@ -2,3 +2,7 @@
 spring.datasource.url = ${spring_datasource_url}
 spring.datasource.username = ${spring_datasource_username}
 spring.datasource.password = ${spring_datasource_password}
+
+#nacos discovery
+nacos.discovery.enabled=false
```

Comment:
To be honest, I don't really like the idea of setting this property in application-github.properties because this is redundant for most of the cases, which uses eureka.

Reply:
Can this configuration be migrated to the apollo-common directory?